core/trie2/triedb/rawdb/reader.go

```diff
@@ -0,0 +1,27 @@
+package rawdb
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/core/trie2/triedb/database"
+	"github.com/NethermindEth/juno/core/trie2/trieutils"
+)
+
+var _ database.NodeReader = (*reader)(nil)
+
+type reader struct {
+	id trieutils.TrieID
+	d  *Database
+}
+
+func (r *reader) Node(
+	owner *felt.Felt,
+	path *trieutils.Path,
+	hash *felt.Felt,
```

Comment:
Are this type on their more descriptive form? Because this is satisfying an interface, then please leave a todo over the interface definition to update the types to more descriptive ones.